ledger_device_sdk/.cargo/config.toml

```diff
@@ -14,8 +14,8 @@ runner = "speculos --model stax --display=headless"
 runner = "speculos --model flex --display=headless"
 
 [unstable]
-build-std = ["core"]
+build-std = ["core", "alloc"]
 build-std-features = ["compiler-builtins-mem"]
 
 [build]
-target = "nanosplus"
+target = "stax"
```

Comment:
Why this change ? :thinking:

Reply:
This sets the default target to build for: https://doc.rust-lang.org/cargo/reference/config.html#buildtarget if we call `cargo build` without `--target`. Any target will fit, was more simple to use `stax` as the default one for my tests...

Reply:
And stax + 🦀  is the future 🚀